component/src/main/resources/rdbms-table-config.xml

```diff
@@ -130,6 +169,13 @@
         <recordUpdateQuery>UPDATE {{TABLE_NAME}} SET {{COLUMNS_AND_VALUES}} {{CONDITION}}</recordUpdateQuery>
         <recordDeleteQuery>DELETE FROM {{TABLE_NAME}} {{CONDITION}}</recordDeleteQuery>
         <recordContainsCondition>({{COLUMNS}} LIKE {{VALUES}})</recordContainsCondition>
+        <selectQueryTemplate>
+            <selectClause>SELECT {{SELECTORS}} FROM {{TABLE_NAME}}</selectClause>
+            <whereClause>WHERE {{CONDITION}}</whereClause>
+            <groupByClause>GROUP BY {{COLUMNS}}</groupByClause>
+            <havingClause>HAVING {{CONDITION}}</havingClause>
+            <orderByClause>ORDER BY {{COLUMNS}}</orderByClause>
```

Comment:
Why there is not offset for some of the queries ?

Reply:
Fixed in 2e64e68